media/css/m24/components/navigation-refresh.scss

```diff
@@ -489,6 +497,16 @@
 }
 
 // page content
+body:has(.c-navigation-items.mzp-is-open) {
+    #outer-wrapper {
```

Comment:
**suggestion:** This is not something we need to solve for now, but `#outer-wrapper` is specific to bedrock and not necessarily something that would be part of Protocol. Perhaps add a comment here for future dev's that this is something we'll need to consider documenting later on?